src/components/Atmosphere.jsx

```diff
@@ -9,12 +9,22 @@ class Atmosphere extends React.Component {
         this.state = {
             rotating: false,
             angle: 0,
+			angleOffset: 0,
         };
-        this.radius = 600;
+		this.radius = 600;
         this.ref = React.createRef();
-        this.handleRotate = _.throttle(this.handleRotate.bind(this), 50);
     }
 
+	componentDidMount() {
+		document.addEventListener('mouseup', this.handleMouseUp);
+		document.addEventListener('mousemove', this.handleRotate);
+	}
+
+	componentWillUnmount() {
+		document.removeEventListener('mouseup', this.handleMouseUp);
+		document.removeEventListener('mousemove', this.handleRotate);
+	}
```

Comment:
Why is this better?

Reply:
![image](https://github.com/plojyon/sol/assets/37750012/e77cba5b-2efa-4be5-ab00-76f32fe9a8ac)

Reply:
did not read :skull: